packages/driver/types/internal-types.d.ts

```diff
@@ -61,13 +89,21 @@ declare namespace Cypress {
     state: Cypress.state
```

Comment:
Can we add the new multi-domain state types? autOrigin, crossOriginBaseUrl, etc.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/6850b688937b82eb48ccd6cde3f6daae384e7a17